### PR TITLE
Phpstan fixes

### PR DESCRIPTION
**Pull request by sreichel**
### Description (*)

A few fixes.

### Contribution checklist (*)
 - [x] Pull request has a meaningful description of its purpose
 - [x] All commits are accompanied by meaningful commit messages
 - [x] All automated tests passed successfully (all builds are green)
 - [x] Add yourself to contributors list

### CODE REVIEW

File: app/code/core/Mage/Sales/Block/Adminhtml/Billing/Agreement/View/Tab/Info.php [hunk 1]
```diff
@@ -107,12 +109,10 @@ protected function _toHtml()
         );
         $this->setCustomerEmail($customer->getEmail());
         $this->setStatus($agreement->getStatusLabel());
-        $this->setCreatedAt(
-            $this->helper('core')->formatDate($agreement->getCreatedAt(), 'short', true)
-        );
+        $this->setCreatedAt($helper->formatDate($agreement->getCreatedAt(), 'short', true));
```
Comment (kiatng) [inline app/code/core/Mage/Sales/Block/Adminhtml/Billing/Agreement/View/Tab/Info.php:112]: Not sure if it's better:
```suggestion
        $this->setCreatedAt($this->formatDate($agreement->getCreatedAt(), 'short', true));
```

File: app/code/core/Mage/Sales/Block/Adminhtml/Billing/Agreement/View/Tab/Info.php [hunk 2]
```diff
@@ -107,12 +109,10 @@ protected function _toHtml()
...
         $this->setUpdatedAt(
             ($agreement->getUpdatedAt())
-                ? $this->helper('core')->formatDate($agreement->getUpdatedAt(), 'short', true) : $this->__('N/A')
+                ? $helper->formatDate($agreement->getUpdatedAt(), 'short', true) : $this->__('N/A')
```
Comment (kiatng) [inline app/code/core/Mage/Sales/Block/Adminhtml/Billing/Agreement/View/Tab/Info.php:115]:
```suggestion
              ? $this->formatDate($agreement->getUpdatedAt(), 'short', true) : $this->__('N/A')
```

**Review by kiatng — COMMENTED** [review]
+1 for `?:` and `??`

File: app/code/core/Mage/Sales/Block/Billing/Agreements.php [hunk 3]
```diff
@@ -86,20 +86,20 @@ public function getItemValue(Mage_Sales_Model_Billing_Agreement $item, $key)
...
                 $label = $item->getAgreementLabel();
-                $value = ($label) ? $label : $this->__('N/A');
+                $value = ($label) ?: $this->__('N/A');
                 break;
             case 'status':
                 $value = $item->getStatusLabel();
                 break;
             default:
-                $value = ($item->getData($key)) ? $item->getData($key) : $this->__('N/A');
+                $value = ($item->getData($key)) ?: $this->__('N/A');
```
Comment (kiatng) [inline app/code/core/Mage/Sales/Block/Billing/Agreements.php:102]: Useless parenthesis:
```suggestion
                $value = $item->getData($key) ?: $this->__('N/A');
```

**Review by kiatng — COMMENTED** [review]
Sorry, didn't spot it earlier.

**Review by kiatng — APPROVED** [review]

**Review by fballiano — APPROVED** [review]

**Pull request merged**